app/src/components/ui/organisms/NavBar/styles.css

```diff
@@ -0,0 +1,11 @@
+.navbar__list {
+  @apply m-2 flex flex-col gap-5 text-xl;
+}
+
+.navbar__disabled-element {
```

Comment:
If the main class is called `navbar__list`, wouldn't it be better for this ones to be called `navbar__list--disabled` and `navbar__list--enabled`?